### PR TITLE
Implement get for RTDB queries

### PR DESCRIPTION
**Pull request by IanWyszynski**
[go/rtdb-get](go/rtdb-get)

### CODE REVIEW

**Comment by IanWyszynski** [comment]
@schmidt-sebastian @paulb777 This PR was failing CI checks for two reasons:

1. The added tests interfered with other tests in the suite (goOnline/goOffline) is required ([code](https://github.com/firebase/firebase-ios-sdk/pull/7264/files#diff-3a8cb26bda8f6a6d6700b7c2ee8b1368a1c147ed7bb29743a0f6146c08d7b723R3082-R3096))
2. The CI in firebase-ios-sdk was still using 4.7.2 (this was addressed in https://github.com/firebase/firebase-ios-sdk/pull/7250)
3. Per Paul's recommendation, I added `scripts/run_database_emulator.sh` to the github workflow paths so that emulator version bumps trigger CI checks ([pr](https://github.com/firebase/firebase-ios-sdk/pull/7268))

The only change here is the `goOnline`/`goOffline` handling. Sebastian, could you have a look and rubberstamp?

File: FirebaseDatabase/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # v7.2.0
 - [added] Made emulator connection API consistent between Auth, Database, Firestore, and Functions (#5916).
+- [added] Added `DatabaseQuery#getData` which returns data from the server when cache is stale (#7110).
```
Comment (paulb777) [inline FirebaseDatabase/CHANGELOG.md:3]: This should be in the 7.5.0 section now.

**Review by paulb777 — APPROVED** [review]
LGTM with a changelog nit

File: FirebaseDatabase/Tests/Integration/FIRDatabaseQueryTests.m [hunk 2]
```diff
@@ -3079,15 +3079,20 @@ - (void)testOfflineQueryGet {
...
-  }];
+    [ref getDataWithCompletionBlock:^(NSError* err, FIRDataSnapshot* snapshot) {
+      XCTAssertNotNil(err);
+      XCTAssertEqualObjects([err localizedFailureReason], kPersistentConnectionOffline);
+      done = YES;
+    }];
 
-  WAIT_FOR(done);
+    WAIT_FOR(done);
+
```
Comment (schmidt-sebastian) [inline FirebaseDatabase/Tests/Integration/FIRDatabaseQueryTests.m:3092]: Nit: Remove empty line.

File: FirebaseDatabase/Tests/Resources/GoogleService-Info.plist.old [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (schmidt-sebastian) [inline FirebaseDatabase/Tests/Resources/GoogleService-Info.plist.old:1]: Please remove.

**Review by schmidt-sebastian — APPROVED** [review]

**Pull request merged**